stripe-ruby-mock.gemspec

```diff
@@ -24,4 +24,5 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'rspec', '~> 3.1.0'
   gem.add_development_dependency 'rubygems-tasks', '~> 0.2'
   gem.add_development_dependency 'thin', '~> 1.6.4'
+  gem.add_development_dependency 'pry'
```

Comment:
Forgot to remove that before pushing. Removed and committed :) Good catch @alexmamonchik !